### DISCUSSION TITLE
news bulletins

### DISCUSSION THREAD

**Discussion by yusufgaf** [AzuraCast/AzuraCast#5182] [Ideas]
Hi 

can you add this to azracast where news where Play news bulletins on the hour every hour from your preferred news provider.

**Comment by LordFader** [comment]
@yusufgaf  maybe the next 2 links could help you.

[Azuracast Suggest Features](https://features.azuracast.com/)
[News Bulletins](https://features.azuracast.com/suggestions/173514/news-bulletins)

**Comment by eternityready2** [comment]
@yusufgaf  @LordFader 

I  updated my version of azura cast with hourly news from AI Newscaster. That comes on air gives news. You get a ui and decide what sites it comes from. You do need a dev background to maintain it. https://github.com/eternityready2/AzuraCast
